File: src/main/java/com/ups/UPSTrailerTracker/TrailerService.java
package com.ups.UPSTrailerTracker;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class TrailerService {
    @Autowired
    private TrailerRepository trailerRepository;

    public void addTrailer(Trailer trailer) {
        trailerRepository.save(trailer);
    }

    public List<Trailer> getTrailers(){
        return (List) trailerRepository.findAll();
    }
}

File: src/main/kotlin/com/ups/UPSTrailerTracker/TrailerService.kt
package com.ups.UPSTrailerTracker

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class TrailerService {
    @Autowired
    lateinit var trailerRepository: TrailerRepository

    fun addTrailer(trailer: Trailer) {
        trailerRepository.save(trailer)
    }

    fun getTrailers(): List<Trailer> = trailerRepository.findAll()
}